glossary.md

```diff
@@ -39,3 +39,19 @@ USA -- typeOf ------------> Country
 USA -- name --------------> United States of America
 USA -- containedInPlace --> northamerica
 ```
+
+- **[Measurement method](https://datacommons.org/browser/measurementMethod)**
+
+The technique used for measuring a statistical variable. Describes how a measurement is made, whether by count or estimate or some other approach, and names the group making the measurement. Examples include `BosniaCensus` and [`WorldHealthOrganizationEstimates`](https://datacommons.org/browser/WorldHealthOrganizationEstimates).
+
+- **[Observation period](https://datacommons.org/browser/observationPeriod)**
+
+The time period over which an observation is made.
+
+- **[Unit](https://datacommons.org/browser/unit)**
+
+The unit of measurement. Examples include Kelvin, Celsius, inches, light years, and slugs.
+
+- **[Scaling factor](https://datacommons.org/browser/scalingFactor)**
+
+Property of statistical variables indicating factor by which a measurement is multiplied to fit a certain format. For example, a proportion of 0.05 displayed as 5% has a scaling factor of 100, since 5 is equal to 0.05 multiplied by 0.05.
```

Comment:
I took a stab-- let me know if this is confusing. Please feel free to make it better, too.

Property of statistical variables, used in conjunction with the measurementDenominator property. Indicates the multiplication factor applied to the denominator when the numerator:denominator ratio is not 1:1. For example, say the percentage of population that is white (statistical variable Count_Person_White_AsAFractionOf_Count_Person) is 5%. We can model the measured value as 5, with the denominator (Count_Person) scalingFactor as 100, since 5% is equal to 5 in 100. Without the denominator, we would interpret the value to be 5/1, or 500%.